.github/ISSUE_TEMPLATE/Document_issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: ⛔ Documentation Issue
+about: See https://github.com/shopware/devdocs/issues for documentation issues
```

Comment:
this is wrong for Shopware 6. the documentation is inside this repository :wink: 

devdocs are Shopware 5 only

Reply:
Hello @mitelg,
So if the document related issues will also be raised in the same repository, then there is no need for this file. I've updated the PR.

Thanks!